tests/unit/records/test_records_manager.py

```
@@ -269,3 +285,268 @@ async def test_profile_results_serialization_with_timeslice(self):
+            self.service_config = service_config
+            self.user_config = kwargs["user_config"]
+            self.service_id = kwargs.get("service_id") or "records_manager"
+            self.pub_client = MagicMock()
+            self.attach_child_lifecycle = MagicMock()
+            self.debug = MagicMock()
+            self.info = MagicMock()
+            self.error = MagicMock()
+            self.exception = MagicMock()
+
+        class DisabledProcessor:
+            def __init__(self, **kwargs) -> None:
```

Ruff (0.15.1)

[warning] Unused method argument: `kwargs`

(ARG002)